xc7/yosys/synth.tcl

```diff
@@ -1,14 +1,22 @@
 yosys -import
+echo on
 
 # -flatten is used to ensure that the output eblif has only one module.
 # Some of symbiflow expects eblifs with only one module.
-synth_xilinx -vpr -flatten
+#
+# Since we do not support BRAMs for now they are not inferred.
+synth_xilinx -vpr -flatten -no-brams
```

Comment:
This is wrong?  BRAM's are supported?

Reply:
Yeah, I've left it by mistake. Corrected.